### PR TITLE
created first draft of SCSS tutorial

### PR DESCRIPTION
**Pull request by Nicknames43**
Adding an introduction to SCSS section to the Tech Stacks part of the repo for my contribution to A2.

### CODE REVIEW

File: Topics/Tech_Stacks/SCSS.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+
+## [Placeholder Selectors and Extend/Inheritance](https://sass-lang.com/documentation/at-rules/extend/)
+
+Placeholder selectors look and act very similar to the CSS class slector. The difference is that they start with a `%` and is not included in the compiled CSS. For example:
+
+#### SCSS
+
+```scss
+%placeholder {
+  /*CSS Block A*/
```
[the remark this answers is not in the document]
Comment (Nicknames43) [reply Topics/Tech_Stacks/SCSS.md]: I made a quick change to the part you highlighted. Let me know if it looks good and if so I'll make similar changes everywhere else.

File: Topics/Tech_Stacks/SCSS.md [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+
+### [Operators](#operators-1)
+
+### [Compiling files to CSS](#compiling-scss-files-into-css-files)
+
+### [References](#references-and-links)
+
+## Introduction
+
+SCSS is a programming language that extends the capabilities of CSS by introducing new syntax to streamline the process of creating stylesheets. SCSS is a superset of CSS which means that all valid CSS code is also valid SCSS code. Due to this, it is recommended that before you start learning SCSS you should have some working knowledge of CSS. You can get started at [this CSS guide](./CSS.md), but it is suggested to have more practice with CSS. Although not required, it is also recomended to use the BEM methodology when creating class names for HTML elements. You can find more information about BEM on [their website here](https://getbem.com/).
```
Comment (SpaceJesusTM) [inline Topics/Tech_Stacks/SCSS.md:29]: This is a solid introduction and I know below you showed the difference between CSS and SCSS but I think it be beneficial to give the user a quick snapshot of the differences between CSS and SCSS and explain why SCSS may be better.

Comment (Nicknames43) [reply Topics/Tech_Stacks/SCSS.md:29]: I have made the changes. Let me know what you think and If I should include more or not.

Comment (SpaceJesusTM) [reply Topics/Tech_Stacks/SCSS.md:29]: I like this better, gives me a sense of why I should care about SCSS/what I could gain from it 👍

File: Topics/Tech_Stacks/SCSS.md [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+## [Compiling SCSS files into CSS files](https://sass-lang.com/guide/#preprocessing)
+
+Once you have created your SCSS files you will need to compile them into CSS files so that your stylesheets can be applied to your website. Some methods to do so include:
+
+1. Run `sass input.scss output.css` in the command line to manually build css files
+2. Run `sass --watch input.scss output.css` in the command line to automatically re-compile input.scss when saved into output.css
+3. Run `sass --watch INPUT_FOLDER_PATH:OUTPUT_FOLDER_PATH` in the command line to automatically re-compile .scss files within INPUT_FOLDER_PATH when saved into a .css file within OUTPUT_FOLDER_PATH
+4. Using the Live Sass Compiler extension in VSCode
+
+## References and Links
```
[the remark this answers is not in the document]
Comment (Nicknames43) [reply Topics/Tech_Stacks/SCSS.md]: I have added 2 video resources and restructured my links section to separate between resources I used to create the guide and additional resources.

**Review by SpaceJesusTM — APPROVED** [review]
Looks good! I have no further changes I think you could make. 

Overall, I liked how you formatted the SCSS vs CSS comparison and gave a rundown of all the various features provided. It gives the guide a very in-depth feel which I think will be useful to anyone looking to speed up their Web Development process. SCSS seems to be under utilized at least from what I've seen so hopefully this helps incentivize people to use it more! 

Good stuff! 🥇

**Pull request merged**